ResearchKit/Common/ORKImageCaptureStep.h

```diff
@@ -65,4 +65,14 @@ ORK_CLASS_AVAILABLE
  */
 @property (nonatomic) UIEdgeInsets templateImageInsets;
 
+/**
+ Accessibility hint of the preview.
+ */
+@property (nonatomic, strong) NSString *accessibilityInstructions;
+
+/**
+ Accessibility hint of the capture button.
+ */
+@property (nonatomic, strong) NSString *accessibilityHint;
+
```

Comment:
Use `copy` instead of `strong` to prevent anyone from passing in a NSMutableString.